src/main.js

```diff
@@ -99,6 +99,9 @@ function deleteIdea(event) {
     if (uniqueID === ideaArray[i].id) {
       ideaArray.splice(ideaArray.indexOf(ideaArray[i]), 1);
     }
+    if (parseInt(uniqueID) === ideaArray[i].id) {
+      ideaArray[i].deleteFromStorage();
```

Comment:
It was great to add this if statement on an already existing function, that way we didn't repeat ourselves!